docs/build-hibench.md

```diff
@@ -48,3 +53,19 @@ Supported modules includes: micro, ml(machine learning), sql, websearch, graph,
 For Spark 2.0 and Spark 2.1, we add the benchmark support for Structured Streaming. This is a new module which cannot be compiled in Spark 1.6. And it won't get compiled by default even if you specify the spark version as 2.0 or 2.1. You must explicitly specify it like this:
 
     mvn -Psparkbench -Dmodules -PstructuredStreaming clean package 
+
+### Build using JDK 1.8
+If you are interested in building using Java 11 specify scala, spark and hadoop version as below
+
+    mvn clean package -Dhadoop=3.2 -Dspark=2.4 -Dscala=2.11
+
+Supported frameworks only: hadoopbench, sparkbench, (Not yet tested flinkbench, stormbench, gearpumpbench)
+Supported modules includes: micro, ml(machine learning), websearch and graph (not tested streaming and structuredStreaming) (Does not support sql)
+
+### Build using JDK 1.11
+If you are interested in building using Java 11 indicate that streaming benchmarks won't be compiled also, specify scala, spark and hadoop version as below
```

Comment:
Should we also specify:
- Which scala version(besides 2.12) is compatible with JDK11?
- Which Hadoop/Spark version is compatible with JDK11?
About the streaming benchmarks support, I think it's okay to lack some streaming(Flink, Gearpump, Spark Streaming, but not Structured Streaming) support on new versions, as long as we pointed it out clearly.

Reply:
**Q:** Which Hadoop/Spark version is compatible with JDK11?
**A:** This is not my area, but documentation could be more specific if required.
At least it can be wrote down that Scala2.12 + JDK11 + Spark2.4 _(Compiled with Scala 2.12)_ works excluding the streaming and SQL benchmarks)_

Reply:
Yes, please indicate that.

Reply:
Done